examples/eks_private_cluster/README.md

```diff
@@ -0,0 +1,8 @@
+## EKS and CAST.AI setup example
+
+Reference setup for EKS clusters without public API access, connected to CAST.AI
```

Comment:
Maybe simpler `Reference setup for EKS clusters connected to CAST.AI`

There is no need to mention network topology as we don't care if we have api access.

Reply:
thats true. initially we had a separate example for eks without controller. I think I'll do that with next PR though